packages/main/src/features/InputSuggestions.js

```diff
@@ -481,14 +481,14 @@ class Suggestions {
 		let text = suggestion.text || suggestion.textContent;
 		text = this.sanitizeText(text);
 
-		return this.hightlightInput(text, input);
+		return this.hightlightInput(text, this.sanitizeText(input));
 	}
 
 	getHighlightedDesc(suggestion, input) {
-		let text = suggestion.description;
+		let text = suggestion.description || "";
 		text = this.sanitizeText(text);
 
-		return this.hightlightInput(text, input);
+		return this.hightlightInput(text, this.sanitizeText(input));
```

Comment:
This won't produce the expected results 100% of the time. There was a reason I had logic in the function that I used before.

```
// Just for demo we set these values explicitly
let text = "&";
let input = "amp"

text = this.sanitizeText(text); // text = "&amp;"
return this.highlightInputtext, this.sanitizeText(input)); // return "&<b>amp</b>;"
```

In the above example you're breaking up HTML entities in half and producing unexpected results.